### PR TITLE
Added UNIX timestamp converter

### PR DESCRIPTION
**Pull request by pranav6670**
# Description

Wrote a short program that uses ```time``` module to convert a timestamp to human interpretable format(DD/MM/YYYY HH:MM).

Fixes #(46)

Replace `issue_no` in the above line, with the issue related to this PR.

## Type of change

Choosing one or more options from the following as per the nature of your Pull request.
- NOTE: *These boxes can be checked using **`[X]`***

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Documentation Update

# Checklist:
Please tick all the boxes that are fulfilled by your Pull Request.

- [X] I have named my files and folder, according to this project's guidelines.
- [X] My code follows the style guidelines of this project.
- [X] My Pull Request has a descriptive title. (not a vague title like `Update index.md`)
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [X] I have created a helpful and easy to understand `README.md`, according to the given [`README_TEMPLATE.`](https://github.com/Python-World/Python_and_the_Web/blob/master/README_TEMPLATE.md)
- [ ] I have included a requirements.txt file (if external libraries are required.)
- [X] My changes do not produce any warnings.
- [X] I have added a working sample/screenshot of the script.

### CODE REVIEW

**Comment by pranav6670** [comment]
Please check my submission! Thank You!

**Review by AdityaJ7 — CHANGES_REQUESTED** [review]
Please remove the DS_Store and .idea files

**Comment by pranav6670** [comment]
> Please remove the DS_Store and .idea files

Oops! Will do it ASAP!

File: Scripts/Miscellaneous/Unix_to_Human/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+### Prerequisites
+```time``` Module
+* The time module is inbuilt with Python. 
+
+### How to run the script
+First you have to go to the Unix_to_Human directory. Run the following command once you are in project directory
+
+```cd Scripts/Miscellaneous/Unix_to_Human```
+
+For Python 3: ```python3 wifiSpeed.py```
```
Comment (chavarera) [inline Scripts/Miscellaneous/Unix_to_Human/README.md:13]: execution file name is different than your pushed code
```suggestion
For Python 3: ```python3 timestamp_to_human.py```
```

File: Scripts/Miscellaneous/Unix_to_Human/timestamp_to_human.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import time
+
+unix_timestamp = int(input())
+utc_time = time.gmtime(unix_timestamp)
```
Comment (chavarera) [inline Scripts/Miscellaneous/Unix_to_Human/timestamp_to_human.py:4]: before passing the UNIX timestamp object validate timestamp

File: Scripts/Miscellaneous/Unix_to_Human/timestamp_to_human.py [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+print(time.strftime("%m/%d/%Y %I:%M%p (UTC)", utc_time))
```
Comment (chavarera) [inline Scripts/Miscellaneous/Unix_to_Human/timestamp_to_human.py:5]: do not put hard-coded values
`UTC`

**Review by chavarera — CHANGES_REQUESTED** [review]
please remove raw and temp file 

add **.idea,DS_Store** in `.gitignore`

**Comment by pranav6670** [comment]
Will resolve all the issues!

File: Scripts/Miscellaneous/Unix_to_Human/timestamp_to_human.py (hunk 2 above)
Comment (pranav6670) [reply Scripts/Miscellaneous/Unix_to_Human/timestamp_to_human.py:4]: Can you please elaborate this? Thank you.

**Comment by pranav6670** [comment]
Hello

**Pull request closed without merging**